code/datums/uplink/revolution.dm

```diff
@@ -0,0 +1,32 @@
+/datum/uplink_item/item/revolution
+	category = /datum/uplink_category/revolution
+
+/datum/uplink_item/item/revolution/armory
+	name = "Armory Dropper"
+	desc = "A device that can be used to drop in an armory-worth of guns. Can only be used outside station areas, unless emagged, which is hazardous."
+	item_cost = 20
+	path = /obj/item/device/orbital_dropper/armory/syndicate
+
+/datum/uplink_item/item/revolution/mecha
+	name = "Mecha Dropper"
+	desc = "A device that can be used to drop in a combat exosuit. Can only be used outside station areas, unless emagged, which is hazardous."
+	item_cost = 25
```

Comment:
The price needs to be increased, so its in line with the rest of the syndie items.